### ISSUE TITLE
Bug: App only supports binary genders

### ISSUE THREAD

**Issue by peppergrayxyz** [simonoppowa/OpenNutriTracker#80]
The app only supports two genders and also assumes all people of these genders have the same calorie formula.

Suggestion: 
- Add gender option "other"
- implement https://github.com/simonoppowa/OpenNutriTracker/issues/61 as a slider
- Indicate average female and male calories on the slider

**Comment by bbb651** [comment]
Correct me if I'm wrong, but I think for calorie tracking what matters is *biological sex* rather *gender*? I think it should be renamed to reflect that, but I don't think "other" makes much sense, maybe a "prefer not to tell". I agree that just having a slider is better though, maybe it should show up only after you select an option, already set on the default value, to keep the interface simple for those who don't care to touch the slider.

**Comment by peppergrayxyz** [comment]
The label in the app says gender. Changing the label to 'biological' sex, wouldn't help without specifying what you mean (e.g. gonads, sexual organs, chromosomes, hormones). Your caloric demand mostly depends on hormone levels, which is different for different people. But also other factors, e.g. on menstrual cycle. The model calculation middles these values out for endo cis people. But I don't think over engineering data entry would help. The App just shouldn't push labels on people they don't use.

From a usability point of view, I think the default calculation makes sense, but data entry could be generally optional/skippable. I think the slider should always show up with the value being set based on the data that was entered. If no data is being entered, you get the demand of an average person. This would neatly integrate in the demand for custom calorie goals. If you already know your calorie goal, you don't have to enter data altogether, but just move the slider.

**Comment by fabian-kir** [comment]
I can understand why this might be an issue for a production version, however for no I think this issue should not be prioritized as most people, and even those who are affected, will probably understand why this information is important and can estimate which choice would better suite them. Maybe it would be enough for no to give an explanaition below:

"! If none of the conventional gender definitions apply to you, please choose the one that comes closest biologically. We would like to point out that we do not want to discriminate or exclude anyone by this selection. We are working on increasing the number of choices."

Maybe a member of the nonbinary-community wants to rewrite this notice, so that no one is being discrimnated!

**Comment by peppergrayxyz** [comment]
Yeah, something like that could be a quick fix! Often gender related things can be fixed, by being more specific, hence I think adding a description is a great idea. Since the underlying issue is reference values, I would actually propose to explain this fact, i.e. something along the lines "Reference values for the basal metabolic rate (BMR) calculator are currently only available for cisgender individuals. Please choose what comes closest to your metabolism."

With that in mind, I'd propose to rename the field from "Gender" to "Metabolism". Since we don't have any additional information to compute the value for non-cisgendered people, the label for a third option could be "average", i.e. just take the average between the male and female reference value. The part with "We are working on increasing the number of choices" is something we can't promise I think, because I'm afraid there is currently no calculator for non-cisgendered people and even if there was, I'm unsure people would be aware of the data it needed to be fed with. So they'd have to go with a custom calorie target anyway.

I think this would fix the issue also long term with minimal changes, mostly by changing the semantics of the field without changing the implementation / algorithm. Most people probably won't care or even notice, that the label is called Metabolism instead of Gender, but for those outside the gender binary I think it's a subtle but import differentiation.

**Comment by lee0c** [comment]
context: hello, trans/nonbinary software dev here! got pointed to this by someone seeking trans input.

> Your caloric demand mostly depends on hormone levels, which is different for different people.

This is key yes! "Biological sex" (besides being a somewhat incoherent concept as @peppergrayxyz demonstrated) is not the most important factor: hormones and specifically your sex hormones are going to be the most reliable indicator of what metabolic profile you should be compared against.

## things i would do

> With that in mind, I'd propose to rename the field from "Gender" to "Metabolism"

In that vein, I would suggest something like:

- **title:** metabolic profile
- **description:** this sets the dataset(s) used for comparison. choose what is most relevant for your body.
- **options:**
  - male average (could also be termed testosterone-dominant, but this may lose some cis people)
  - female average (could also be termed estrogen-dominant, but same note as above)
  - all average (include all data as suggested above)

I would also strongly suggest the calorie slider mentioned as well, as it supports and extends this type of view.

## things i would NOT do

- use the term "biological sex" or "birth sex" - as mentioned this is not the most relevant factor
- say things like "we do not wish to discriminate" - you can convey this more effectively by offering non-discriminatory choices!
- promise outcomes that you cannot or will not deliver - as Pepper pointed out, good will alone does not create health datasets out of thin air :)
- tie this to "gender" in any other way - no pink and blue, no calling it gender, nothing. neutral field.

**Comment by kxmpxtxnt** [comment]
I‘m also working on such a app, i came to the same problem. My solution, given by an subreddit was to ask if the users body is alle to menstruate.

**Comment by Splarkszter** [comment]
People just threw hate on this issue without even reading.

The user provided a good solution and alternative:
>     * Add gender option "other"
> 
>     * implement [Custom Calorie goals #61](https://github.com/simonoppowa/OpenNutriTracker/issues/61) as a slider
> 
>     * Indicate average female and male calories on the slider

Hormone treatments will change your metabolism, and every person is different.
Don't be assholes.

**Comment by droped** [comment]
There are no equations for the ERR of trans individuals so any other genders would break half of the features in the app. This is a completely unreasonable and frankly impossible request regardless of the supposed "incoherence" of biological sex.

**Comment by droped** [comment]
>things i would do

You're asking to make the app more complicated for everyone for the sake of a small minority that wouldn't even understand the changes you're making (it's not like trans people are born knowing what "metabolic profile" or "metabolism" means).

This is like asking to make the app more inclusive of people who lost a body part. There simply aren't enough studies to calculate the general ERR of those people.

**Comment by jordan-lee-code** [comment]
Hi @peppergrayxyz — confirming this is still outstanding on `develop`.

Code location: `lib/core/domain/entity/user_gender_entity.dart:5-7` declares only `male` and `female`; the IOM-2005 calorie formula at `lib/core/utils/calc/tdee_calc.dart:27-51` branches on those two values, so a third entry needs both an enum addition (mind the Hive `typeId` on `UserGenderDBO`) and a calorie-formula default (probably an average of the male / female constants, or expose a kcal slider).

Labelled `enhancement`. Contributor PRs welcome.

**Comment by jordan-lee-code** [comment]
**Design proposal — non-binary inclusivity (2026-05-06)**

Following up on the implementation note above with a more thoughtful design that includes a non-binary option without forcing users to disclose anything personal.

**Approach: `Non-binary / Other` enum entry, with a private optional hormone-profile setting for users who want a more accurate calorie starting point.**

### Data model

Decouple identity from calorie-math by adding two separate fields rather than overloading one:

```dart
// UserGenderEntity (lib/core/domain/entity/user_gender_entity.dart)
enum UserGenderEntity {
  male,
  female,
  nonBinary,  // new
}

// CaloriesProfileEntity (new file — lib/core/domain/entity/calories_profile_entity.dart)
enum CaloriesProfileEntity {
  averaged,            // default for nonBinary; mean of male / female reference values
  estrogenTypical,     // uses the female reference formula (e.g. trans women on HRT)
  testosteroneTypical, // uses the male reference formula (e.g. trans men on HRT)
}
```

Why two fields: a user's self-identification (gender) and the calorie-math reference (which physiology the formula was derived from) are different things. Coupling them forces non-binary users to pick a binary identity to get accurate calories, which is exactly what this issue is asking us to avoid.

### Default behaviour

- When `gender == nonBinary` and `caloriesProfile` is unset → use the **averaged** formula. This is the mean of the IOM 2005 male and female constants and gets users a reasonable starting number with zero further questions.
- For `male` / `female` selections, `caloriesProfile` stays unset and the existing formulas at `lib/core/utils/calc/tdee_calc.dart:27-51` continue to apply unchanged.

### Optional disclosure (info icon)

Next to the gender selector during onboarding (and again in Settings → Calculations), surface an info icon that opens a small dialog:

> "We use an averaged calorie reference for non-binary users by default. If you're on HRT or want a more accurate starting point, you can set your hormone profile here. This affects only the calorie calculation — it isn't shared anywhere or shown elsewhere in the app."
>
> **Hormone profile** (optional)
> - Use averaged reference (default)
> - Estrogen-typical reference
> - Testosterone-typical reference

The disclosure is opt-in. The wording emphasises that it's about the math, not identity, and that the data stays local (true — it's all in the AES-encrypted Hive store).

### Calibration mechanism

The existing ±1000 kcal slider in Settings → Calculations (#313) is the user-facing fine-tune. Whatever starting number the formula produces, users can self-correct without the app needing to ask further personal questions.

### Implementation surface

1. `UserGenderEntity` — add `nonBinary` value (mind `UserGenderDBO` Hive `typeId`)
2. New `CaloriesProfileEntity` enum + `CaloriesProfileDBO`
3. New nullable field `UserDBO.caloriesProfile` (next free `@HiveField`)
4. `TDEECalc.getTDEEKcalIOM2005` — branch on gender + caloriesProfile (averaged = mean of male/female outputs; estrogenTypical → existing female branch; testosteroneTypical → existing male branch)
5. Onboarding gender page + Settings → Calculations — gender option + info-icon disclosure
6. l10n strings for the new option and the disclosure dialog
7. `hive_registrar.g.dart` regen

Estimated scope: ~1 day of focused work plus translation passes for the 8 supported locales.
